tools/postman/validation/spectral.yaml

```diff
@@ -0,0 +1,89 @@
+functions:
+  [
+    folderDescription,
+    requestNameShouldNotEndInCopy,
+    patchRequestShouldHaveBody
+  ]
+
+rules:
+  collection-description-is-mandatory:
+    description: Collection must have a description.
+    given: $.collection.info
+    severity: error
+    then:
+      field: description
+      function: truthy
+
+  request-description-is-mandatory:
+    description: All requests must have a description.
+    given: $..request
+    severity: error
+    then:
+      field: description
+      function: truthy
+
+  folder-description-is-mandatory:
+    description: All folders should have a description.
+    message: "{{error}}"
+    given: $
+    severity: warn
+    then:
+      function: "folderDescription"
+
+  request-url-is-mandatory:
+    description: Requests must have a URL.
+    given: $..request
+    severity: error
+    then:
+      field: url
+      function: truthy
+
+  all-responses-have-status-code:
+    description: All responses must have a status code.
+    given: $..response[*]
+    severity: error
+    then:
+      field: code
+      function: truthy
+
+  http-should-not-be-used:
+    description: HTTP should not be used.
+    message: "HTTP should not be used: {{value}}"
+    given: $..url.raw
+    severity: warn
+    then:
+      function: pattern
+      functionOptions:
+        notMatch: "/^http:/i"
```

Comment:
I believe this comes out of the box if you include the default rules https://github.com/mongodb/openapi/blob/0819d1d3d007c4498e10db1518b977b873bdd28b/.spectral.yaml#L2-L3

Reply:
I didn't see a rule in here that corresponds so I will keep it for now
https://docs.stoplight.io/docs/spectral/4dec24461f3af-open-api-rules#openapi-rules